app/assets/stylesheets/application.css.scss

```diff
@@ -255,8 +278,24 @@ main {
     top: 9px;
     right: 2px;
     position: absolute;
+    border-radius: 5px;
+    color: white;
+    background-color: black;
+    border: none;
+}
+
+.avatar-selection{
```

Comment:
Merge rule `.avatar-selection` with rule on line 173<br>Opening curly brace `{` should be preceded by one space